### DISCUSSION TITLE
Contributing realted

### DISCUSSION THREAD

**Discussion by codecdekamal** [wagtail/gsoc#26] [Q&A] [answered]
How I can contribute to your Project.

**Comment by laymonage** [comment, marked as answer]
Hello, we have a dedicated guide for making contributions to Wagtail at https://github.com/wagtail/wagtail/issues/10070. Please give it a read, thanks!

**Comment by thibaudcolas** [comment]
@codecdekamal at this stage I’d recommend reading through  our [contributor guide](https://github.com/wagtail/gsoc/blob/main/contributor-guide.md) for GSoC participants specifically.
